.devcontainer/.env

```diff
@@ -1,6 +1,13 @@
+# common service settings:
+LC_COLLATE=POSIX
```

Comment:
this one is a database setting!

Reply:
Also please capitalise the comments and choose either `:` or `.` (or nothing) so they're consistent

Reply:
fixed.